### PR TITLE
feat: change throw error to console.warn and fix #92

### LINKED ISSUE #92: Remove throw new error code when find some props change in componentWillReceiveProps
Throw new error will break the flow in somewhere like ajax.

And I saw the code https://github.com/roadmanfong/react-cropper/blob/master/src/react-cropper.js#L115 just a tips, so it should be a console.warn?

Btw, if I use the react cropper and put any unchangeable props on it, every time my setState invoked, it will invoke this check logic. So it's really necessary?

### PR DESCRIPTION
**Pull request by yujiangshui**

### CODE REVIEW

**Comment by yujiangshui** [comment]
@roadmanfong

**Comment by yujiangshui** [comment]
@roadmanfong 还在维护这个组件么？

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "url": "https://github.com/roadmanfong/react-cropper.git"
   },
   "dependencies": {
-    "cropperjs": "v1.0.0-rc.3",
+    "cropperjs2": "^2.0.0",
```
Comment (Sowed) [inline package.json]: Hello @yujiangshui. I have been trying to lookup [cropperjs2](https://www.npmjs.com/package/cropperjs2), but cannot seem to find the change log from [cropperjs](https://www.npmjs.com/package/cropperjs). How are they different besides the only difference that I can find that cropperjs2 omits [initialAspectRatio](https://www.npmjs.com/package/cropperjs#initialaspectratio) from its options.

I also see you published [react-cropper2](https://www.npmjs.com/package/react-cropper2), but like cropperjs2, the documentation and repo all point back to the previous version without a descriptive change log.
How are these packages different.

Comment (gfx) [reply package.json]: I think @yujiangshui should maintain their own `@yujiangshui/react-cropper`, so I've add -1 to this PR.

**Comment by roadmanfong** [comment]
先保持使用 cropperjs1

**Pull request closed without merging**